telegram/ext/_application.py

```diff
@@ -1227,7 +1227,7 @@ async def __create_task_callback(
                 await self.process_error(update=update, error=exception, coroutine=coroutine)
 
             # Raise exception so that it can be set on the task and retrieved by task.exception()
-            raise exception
+            raise
```

Comment:
So python automatically raises the latest available exception?

Reply:
yep